src/globals.test.js

```diff
@@ -445,4 +453,26 @@ describe('globals', () => {
 
     process.env = { ...oldEnv };
   });
+
+  test.each`
+    killSwitchValue | isAwsEnvironment | expectedRetValue | expectedTags
+    ${false}        | ${true}          | ${true}          | ${[{ key: 'k0', value: 'v0' }]}
+    ${true}         | ${true}          | ${false}         | ${[]}
+    ${true}         | ${false}         | ${false}         | ${[]}
+    ${false}        | ${false}         | ${false}         | ${[]}
+  `(
+    'killSwitch=$killSwitchValue, isAwsEnv=$isAwsEnvironment, expectedRetValue=$expectedRetValue$',
+    ({ killSwitchValue, isAwsEnvironment, expectedRetValue, expectedTags }) => {
+      const oldEnv = Object.assign({}, process.env);
```

Comment:
You don't need to save the envs. the infra do it for you.
Look at steup.js

Reply:
done